src/test/java/core/basesyntax/service/CsvReportServiceTest.java

```diff
@@ -0,0 +1,49 @@
+package core.basesyntax.service;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.impl.CsvReportService;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+public class CsvReportServiceTest {
+    private static ReportService reportService;
+
+    @BeforeAll
+    static void beforeAll() {
+        reportService = new CsvReportService();
+    }
+
+    @Test
+    void generateReport_validData_ok() {
+        Storage.FRUITS.put("banana", 152);
+        Storage.FRUITS.put("apple", 90);
+        String expected = "fruit,quantity" + System.lineSeparator()
```

Comment:
You can create constant for "fruit,quantity" and in that way avoid code duplication

Reply:
+

Reply:
still exists.